guides/api.md

```diff
@@ -97,8 +97,8 @@ defmodule MyAppWeb.APIAuthPlug do
       |> Conn.put_private(:api_auth_token, token)
       |> Conn.put_private(:api_renew_token, renew_token)
 
-    CredentialsCache.put(store_config, token, user)
-    PersistentSessionCache.put(store_config, renew_token, user.id)
+    CredentialsCache.put(store_config, token, {user, []})
+    PersistentSessionCache.put(store_config, renew_token, {[id: id], []})
```

Comment:
Getting an error here:
** (CompileError) lib/spades_web/api_auth_plug.ex:36: undefined function id/0
```
```

Reply:
Updated to `[id: user.id]` thanks!